util/src/main/i18n/templates/moderation.properties

```
@@ -157,6 +157,13 @@ moderation.defuse.world = You defused world TNT.
 # {0} = player name
 moderation.defuse.player = You defused {0}'s TNT.
 
+# {0} = defuser player name
+# {1} = griefer player name
+moderation.defuse.alert.player = {0} defused {1}'s TNT.
```

```suggestion
moderation.defuse.alert.player = {0} defused {1}'s {2}.
```
TNT should be localized aswell, using minecraft's name